pkg/baseline/packageIndex.json

```diff
@@ -113,6 +113,12 @@
         "4.0.3.0": "4.4.0",
         "4.0.4.0": "4.5.0"
       }
+    },
```

Comment:
Note that this is also going to change the package references for your netstandard2.0 dependencies which you might not want. Please verify that it doesn't cause any issues.